.github/workflows/build-old.yaml

```diff
@@ -0,0 +1,323 @@
+name: Build Wolfi OS with make all
+
+on:
+  push:
+    branches: ['main']
```

Comment:
I think we should comment this out, so we don't build everything twice, right?

Reply:
Indeed!